Feature/update docker k8s (#207)
AxonIQ/reference-guide: axon-server/installation/docker-k8s/axon-server-ee.md

* The "`root`" image of version 4.5.3 is available as "`axoniq/axonserver-enterprise:4.5.3-dev`" and is based on "`gcr.io/distroless/java:11-debug`". This image is particularly useful for running in Docker Desktop, as it will not have any trouble creating files and directories as user "`root`".
* The "`axonserver`" image of version 4.5.3 is available as "`axoniq/axonserver-enterprise:4.5.3-dev-nonroot`" and is based on "`gcr.io/distroless/java:11-debug-nonroot`". This image is more secure and useful in Kubernetes and OpenShift clusters. You should take care to declare the user- and group-id, both of which are `1001` and are named "`axonserver`". Doing this will ensure that any mounted volumes will be writable by the user running Axon Server.

> Version 4.5.3 is the first version for which we have released these images, and later versions will follow. Please replace "4.5.3" in this page with the appropriate version number as needed. The images export the following volumes:

Reviewer: ```suggestion
Version 4.5.3 is the first version for which we have released these images, and later versions will follow. Please replace "4.5.3" on this page with the appropriate version number as needed. The images export the following volumes:
```

* "`/axonserver/config`"

>   This where you can add configuration files, such as an additional `axonserver.properties` and the license file, although you can also opt to use e.g. Kubernetes or Docker-compose secrets. Note that Axon Server EE assumes it can write to the directory configured with "`axoniq.axonserver.enterprise.licenseDirectory`", so you don't have to put the license on all nodes.

Reviewer: ```suggestion
  This is where you can add configuration files, such as an additional `axonserver.properties` and the license file. However, you can also opt to use, e.g., Kubernetes or Docker-compose secrets. Note that Axon Server EE assumes it can write to the directory configured with "`axoniq.axonserver.enterprise.licenseDirectory`", so you don't have to put the license on all nodes.
```

## Docker Compose

Axon Server EE is meant to be run in a distributed manner i.e. as a cluster where there will be multiple instances of Axon Server EE nodes running all interconnected to each other.

The installation process assumes that Docker Compose will be used to run a 3-node Axon Server EE cluster i.e. running 3 services of the same container image we built above. Let us designate these services as "_axonserver-1_", "_axonserver-2_" and "_axonserver-3_". We will also give a tag to the image that we constructed above as "_my-repository/axonserver-enterprise:4.5.3_".

Each container instance will use separate volumes for “data”, “events”, and “log”. We will use "secrets" to inject the license file, tokens as well as the cluster/context definitions using the [autocluster](../local-installation/axon-server-ee.md#auto-clustering) mechanism. An environment variable is added to tell Axon Server about the location of the license file.

> The complete docker-compose file is depicted below. Note you have 

Reviewer: Guessing the `Note you have`... part is a left-over/typo?

$ docker-compose up
```

> **Note 1**

Reviewer: Nit: The style of this note is different than the other parts of the RefGuide (which are up to date).
Surely a style preference, but what I've been doing throughout is the following:

```
> **{short description of the note as header}**
>
> {note text}
```

The examples below show only one of the ways you could deploy Axon Server to Kubernetes. As discussed in [this Blog article](https://developer.axoniq.io/w/revisiting-axon-server-in-containers), there are many aspects that you need to carefully plan ahaead for. A more complete set of examples can be found in the "[Running Axon Server](https://github.com/AxonIQ/running-axon-server)" GitHub repository. We especially recommend using [the "Singleton StatefulSet" approach](https://github.com/AxonIQ/running-axon-server/tree/master/3-k8s/4-k8s-ee-ssts-tls).

**Note 2**

> Although the complexity of deploying any application to Kubernetes can be overwhelming, we strongly recommend you to study this subject carefully. The examples we provide are not necessarily the best approach for your particular situation, so be careful about copying them without any further modifications, if only because they generate self-signed certificates that have only a one-year validity.

Reviewer: The double use of the word 'only' feels a little off here:

```suggestion
Although the complexity of deploying any application to Kubernetes can be overwhelming, we strongly recommend you to study this subject carefully. The examples we provide are not necessarily the best approach for your particular situation, so be careful about copying them without any further modifications, if only because they generate self-signed certificates that have a one-year validity.
```
